addons/mail/static/tests/message/message.test.js

```diff
@@ -2029,3 +2029,28 @@ test("deleted message should not have translate feature", async () => {
         await contains(".dropdown-item:contains('Translate')", { count: 0 });
     }
 });
+
+test("should delete link preview along with message", async () => {
```

Comment:
This only tests the mock server. Could you please write a test in python instead?